### ISSUE TITLE
Bbox detection problem on Test set 

### ISSUE THREAD

**Issue by MohammedAzizi** [kevinjliang/tf-Faster-RCNN#23]
Hi ,

I am using the 5-th layer model on my own DataSet .. but after some epochs ( 20 ) , I checked the test result , then for all the test set ( 90 images ) I had a same 'strange' result : No correct detection and only the whole image box appears as detected

http://hpics.li/5ab74fb 

I am using a grayscale images.

Could you help to fix that problem.

Thank you 

Mohammed

**Comment by kevinjliang** [comment]
Do you have the latest commit? How large is your dataset?

**Comment by MohammedAzizi** [comment]
Hi Kevin,

I have the latest commit , 

 the dataset is approximately 600images with 6/7 objects per image.

The result is too bad and tuning the learning rate and the weight decay didn't impact the detection performance.

Thanks Kevin

**Comment by kevinjliang** [comment]
That's a fairly small dataset. How big are your images, and how complex are they? What are you trying to detect? It could be that you need a more powerful CNN than the simple 5-layer one I have as a demo.

**Comment by MohammedAzizi** [comment]
Thanks for your answer Kevin

My images was 541x340 , but now i am trying a bigger version of RGB images with high resolution . Something about 3000x2000 with the ResNet model for RPN.

I will be back to tell you about the results after running.

Mohammed
